src/reference/v8.json

```diff
@@ -4568,19 +4591,33 @@
       }
     }
   },
-  "projection": {
+  "projectionConfig": {
     "type": {
-      "type": "enum",
-      "doc": "The projection type.",
+      "type": "projection",
+      "doc": "The projection type. Can be specified as a string, a transition state, or an expression.",
       "default": "mercator",
+      "expression": {
+        "interpolated": true,
+        "parameters": [
+          "zoom"
+        ]
+      },
       "values": {
-        "mercator": {
-          "doc": "The Mercator projection."
+        "projections": {
```

Comment:
I would remove these (projections, presets), as they are proprietary to this type and are not validated or checked, right?
If this is the case, and this is added only for documentation purposes, use the docs with markdown to extend how the docs will look.

Reply:
They are actually validated right now because of

https://github.com/maplibre/maplibre-style-spec/pull/888/files#diff-22cb3d567c884a921b94b7964eecbaa9b22ee8b0be35ba1e7118b484b3c0dadaR129-R130

I can prefer this if it's not wanted.

Reply:
Removed it